nextcloudappstore/core/templates/user/api-token.html

```diff
@@ -2,10 +2,32 @@
 {% load i18n %}
 {% load staticfiles %}
 
+{% block head %}
+    <script defer="defer" src="{% static 'assets/js/utils.js' %}"></script>
+    <script defer="defer" src="{% static 'assets/js/account-api-token.js' %}"></script>
+{% endblock %}
+
 {% block head-title %}{% trans 'API Token' %} - {% endblock %}
 
 {% block account-content %}
     <h1>{% trans "API Token" %}</h1>
-    <section>
+    <section id="tokenSection">
+        <div id="regen-success" class="alert alert-success" hidden>{% trans 'New API token generated.' %}</div>
+        <div id="regen-failure" class="alert alert-danger" hidden>{% trans 'Failed to regenerate API token.' %}</div>
+        <div id="token-failure" class="alert alert-danger" hidden>{% trans 'Failed to fetch token.' %}</div>
+        <span id="regen-confirm-text" hidden>{% trans 'Do you want to regenerate your API token?' %}</span>
+        {% blocktrans %}
```

Comment:
Fixed. But yeah, I think i insist on blocktrans in order to keep the line length manageable :)